### PR TITLE
Connections documentation for k8s

### PR DESCRIPTION
**Pull request by saurabh-jogalekar**

### CODE REVIEW

File: MLOps/examples/KDApp connections/Configmeta.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        "role_id": "controller",
+        "nodegroup_id": "1",
+        "id": "1",
+        "hostname": "kdss-dnlkm-0.kdhs-f2vb7.kubecon.svc.cluster.local",
+        "fqdn": "kdss-dnlkm-0.kdhs-f2vb7.kubecon.svc.cluster.local",
+        "domain": "kdhs-f2vb7.kubecon.svc.cluster.local",
+        "distro_id": "hpecp/jupyter-notebook",
+        "depends_on": {}
+    },
+    "connections": {
```
Comment (kmathur2) [inline MLOps/examples/KDApp connections/Configmeta.md:136]: Lets just put only , and give code example to retrieve a key from data, example proxy-hostname
```
"connections": {
        "clusters": {},
        "configmaps": {
            "source-control": [{
                "annotations": {
                    "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"data\":{\"branch\":\"master\",\"proxy-hostname\":\"web-proxy.corp.hpecorp.net\",\"proxy-port\":\"8080\",\"proxy-protocol\":\"http\",\"repo-url\":\"http://repo:8080/scm/~user/test-repo-sj.git\",\"type\":\"bitbucket\",\"user-email\":\"suser@hpe.com\",\"user-name\":\"saurabhj\"},\"kind\":\"ConfigMap\",\"metadata\":{\"annotations\":{},\"labels\":{\"kubedirector.hpe.com/cmType\":\"source-control\"},\"name\":\"source-control-info\",\"namespace\":\"kubecon\"}}\n"
                },
                "data": {
                    "branch": "master",
                    "proxy-hostname": "web-proxy.corp.hpecorp.net",
                    "proxy-port": "8080",
                    "proxy-protocol": "http",
                    "repo-url": "http://repo:8080/scm/~saurabhj/test-repo-sj.git",
                    "type": "bitbucket",
                    "user-email": "user@hpe.com",
                    "user-name": "user"
                },
                "labels": {
                    "kubedirector.hpe.com/cmType": "source-control"
                },
                "metadata": {
                    "name": "source-control-info"
                }
            }]
        },
        "secrets": {}
    }
}

**Comment by joel-bluedata** [comment]
In illustrating the use of secrets connections, be sure to cover the fact that everything (including metadata/labels/annotations) is scrambled... base64 I think?

**Comment by kmathur2** [comment]
> In illustrating the use of secrets connections, be sure to cover the fact that everything (including metadata/labels/annotations) is scrambled... base64 I think?

Yes, good point !!

**Comment by saurabh-jogalekar** [comment]
> > In illustrating the use of secrets connections, be sure to cover the fact that everything (including metadata/labels/annotations) is scrambled... base64 I think?
> 
> Yes, good point !!

Addressed in the latest commit

**Pull request merged**